x-pack/plugins/cases/public/containers/use_get_cases.tsx

```diff
@@ -212,6 +212,7 @@ export const useGetCases = (
 
   const dispatchUpdateCaseProperty = useCallback(
     async ({ updateKey, updateValue, caseId, refetchCasesStatus, version }: UpdateCase) => {
+      const caseData = state.data.cases.find((el) => el.id === caseId);
```

Comment:
nit: Just curious, what does `el` stand for here? `caseInfo` might be more clear

Reply:
`el` = `element`